src/test/java/edu/unc/lib/boxc/migration/cdm/AggregateFilesCommandIT.java

```diff
@@ -68,6 +68,9 @@ public void generateBasicMatchTopTest() throws Exception {
 
         assertTrue(Files.exists(project.getAggregateTopMappingPath()));
         assertFalse(Files.exists(project.getAggregateBottomMappingPath()));
+        assertOutputMatches(".*New Files Mapped: +0.*");
```

Comment:
hmm, i don't think these mapping counts are right, it seems like two mappings were supposed to be added? I am guessing what might be the issue is that SourceFilesSummaryService initializes a SourceFileService internally, but it needs to be using an AggregateFileService instead in order to get the correct mapping paths. So SourceFilesSummaryService probably needs to be updated to allow its sourceFileService to be injected so that we can pass in the correct service. This would apply to AccessFiles as well, those counts look off too.